openedxscorm/scormxblock.py

```diff
@@ -202,6 +205,33 @@ def student_view(self, context=None):
         )
         return frag
 
+    @XBlock.handler
+    def assets_proxy(self, request, _suffix):
```

Comment:
If I understand your implementation correctly, there is no need to go through the `assets_proxy` when authentication is disabled, right? Can you please add a corresponding comment in this docstring?